# ==============================================================================
# REGRESSOR COMMIT DIFF
# ==============================================================================
# File: layout/generic/nsGfxScrollFrame.h
# Commit: a269192c5d30
# Full Hash: a269192c5d3037c7841d30219516548a6fa8db1a
# Author: Hiroyuki Ikezoe <hikezoe@mozilla.com>
# Date: 2019-06-27 09:34:48
# Regressor Bug: 1560237
# File Overlap Count: 1
# Description:
#   Bug 1560237 - Rename ScrollFrameHelper::GetFrameForScrollSnap to ScrollFrameHelper::GetFrameForStyle. r=botond
#   
#   The function will also be used for `scroll-behavior` property in a later patch
#   in this commit series, so it needs a more reasonable name.
#   
# ==============================================================================

diff -r 065a45b8659d -r a269192c5d30 layout/generic/nsGfxScrollFrame.h
--- a/layout/generic/nsGfxScrollFrame.h	Wed Jun 26 20:59:01 2019 +0000
+++ b/layout/generic/nsGfxScrollFrame.h	Wed Jun 26 20:59:14 2019 +0000
@@ -347,9 +347,13 @@
  private:
   nsIFrame* GetFrameForDir() const;  // helper for Is{Physical,Bidi}LTR to find
                                      // the frame whose directionality we use
-  nsIFrame* GetFrameForScrollSnap() const;  // helper to find the frame whose
-                                            // scroll-snap-type and
-                                            // scroll-padding we use
+  // helper to find the frame that style data for this scrollable frame is
+  // stored.
+  //
+  // NOTE: Use GetFrameForDir() if you want to know `writing-mode` or `dir`
+  // properties. Use GetScrollStylesFromFrame() if you want to know `overflow`
+  // and `overflow-behavior` properties.
+  nsIFrame* GetFrameForStyle() const;
 
   // This is the for the old unspecced scroll snap implementation.
   ScrollSnapInfo ComputeOldScrollSnapInfo() const;
